js/search.js

```diff
@@ -68,13 +76,13 @@ $(document).ready(function() {
 
     let queryTerms = searchTerm.trim().toLowerCase().split(" ");
     if (queryTerms.length === 1){
-      parsedQuery = searchQuery
+      parsedQuery = preProcessQueryTerm(queryTerms[0]);
     } else {
       for (const term of queryTerms) {
         if (stopWords.includes(term)) {
-          parsedQuery += `${term} `
+          parsedQuery += `${preProcessQueryTerm(term)} `
```

Comment:
So I'm a little unclear about this line. Basically, we can't do boolean AND searching on stop words?

Reply:
Also, I guess I missed this in my PR.

Reply:
This is part of #79, which was already merged. As I said there, requiring stop words to be present returns unexpected results, because those words are not present in the index.

Reply:
Ok, that makes sense. I must have not put two and two together on that one. Thanks for the clarification.